web/satellite/src/router/index.ts

```diff
@@ -165,7 +176,28 @@ export const router = new Router({
                     name: RouteConfig.EditProjectDetails.name,
                     component: EditProjectDetails,
                 },
+                {
+                    path: RouteConfig.NameStep.path,
```

Comment:
previous version was correct here. AccessGrant is a parent route of these 4 routes. So routes will be like this 
`/access-grants/access-create-name`. That means that we need AccessGrant component but not in the views folder like I mentioned before because AccessGrant is a child of dashboard but a parent of these 4 routes. Just return previous version of this file but change import path of AceessGrant.vue file

it should be imported like this `import AccessGrant from '@/components/accessGrants/AccessGrant.vue';`